gradle.properties

```diff
@@ -1,3 +1,3 @@
 jdk.version=17
-artifactVersion=2.3.4-SNAPSHOT
-androidAttestationVersion=1.7.3-SNAPSHOT
\ No newline at end of file
+artifactVersion=2.4.1-SNAPSHOT
```

Comment:
why not 2.4.0-SNAPSHOT?

Reply:
oops! that was a leftover from pupil id debugging, thanks!